app/src/main/res/values/strings.xml

```diff
@@ -202,6 +202,12 @@ Tap this message (or hit back) to skip this step.</string>
   <string name="no_description_found">no description found</string>
   <string name="nearby_info_menu_commons_article">Commons file page</string>
   <string name="nearby_info_menu_wikidata_article">Wikidata item</string>
-    <string name="error_while_cache">Error while caching pictures</string>
+  <string name="error_while_cache">Error while caching pictures</string>
+  <string name="feedback_popup_title">We need your feedbacks</string>
+  <string name="feedback_popup_description">We are planning several new features and improvements for the app! Have your say.</string>
```

Comment:
If we are putting the link in the positive action button, we should probably change this to a question. "We are planning several new features and improvements for the app! Would you like to review them and provide feedback?"